manifests/individual_connection.pp

```diff
@@ -0,0 +1,392 @@
+      pki    => $pki,
+      notify =>  Service["stunnel_${name}"]
+    }
+  }
+    
+  if $_chroot {
+    # The _chroot directory
+    file { $_chroot:
+      ensure => 'directory',
+      owner  => 'root',
+      group  => $setgid,
+      mode   => '0770',
+      tag    => 'firstrun',
+    }
+
+    # The following two entries are required to be able to properly resolve
+    # hosts using the _chroot directory.
+    file { "${_chroot}/etc":
+      ensure => 'directory',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0755',
+      tag    => 'firstrun',
+    }
+
+    file { "${_chroot}/etc/resolv.conf":
+      ensure => 'file',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0644',
+      source => 'file:///etc/resolv.conf',
+      tag    => 'firstrun',
+    }
+
+    file { "${_chroot}/etc/nsswitch.conf":
+      ensure => 'file',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0644',
+      source => 'file:///etc/nsswitch.conf',
+      tag    => 'firstrun',
+    }
+
+    file { "${_chroot}/etc/hosts":
+      ensure => 'file',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0644',
+      source => 'file:///etc/hosts',
+      tag    => 'firstrun',
+    }
+
+    file { "${_chroot}/var":
+      ensure => 'directory',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0644'
+    }
+
+    file { "${_chroot}/var/run":
+      ensure => 'directory',
+      owner  => $setuid,
+      group  => $setgid,
+      mode   => '0644'
+    }
+
+    file { "${_chroot}/var/run/stunnel":
+      ensure => 'directory',
+      owner  => $setuid,
+      group  => $setgid,
+      mode   => '0644'
+    }
+
+    file { "${_chroot}/etc/pki":
+      ensure => 'directory',
+      owner  => 'root',
+      group  => $setgid,
+      mode   => '0640'
+    }
+
+    file { "${_chroot}/etc/pki/cacerts":
+      source  => "file://${app_pki_dir}/cacerts",
+      group   => $setgid,
+      mode    => '0640',
+      recurse => true,
+    }
+  }
+
+  # The rules are pulled together from the accept_* and connect_*
+  # variables.
+  #
+  # This is only enabled if the system is a server.
+  if $firewall and !$client {
+    include '::iptables'
+
+    $_dport = [to_integer(split(to_string($accept),':')[-1])]
+
+    iptables::listen::tcp_stateful { "allow_stunnel_${name}":
+      trusted_nets => $trusted_nets,
+      dports       => $_dport
+    }
+  }
+
+  if !$client and $tcpwrappers {
+    include '::tcpwrappers'
+
+    tcpwrappers::allow { "allow_stunnel_${name}":
+      pattern => nets2ddq($trusted_nets)
+    }
+  }
+
+  file { "/etc/rc.d/init.d/stunnel_${name}":
+    ensure  => 'present',
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0750',
+    content => template('stunnel/individual_init.erb'),
+    tag     => 'firstrun',
+    notify  => Exec["stunnel_${name}_chkconfig_update"],
+  }
+
+  exec { "stunnel_${name}_chkconfig_update":
```

Comment:
I'm honestly not sure why we're doing this.

Adding `enable => true` to the `service` statement *should* work.